lib/topologies/server.js

```diff
@@ -392,7 +392,10 @@ Object.defineProperty(Server.prototype, 'port', {
 Object.defineProperty(Server.prototype, 'autoEncrypter', {
   enumerable: true,
   get: function() {
-    return this.s.options.autoEncrypter;
+    if (server.s && server.s.options && server.s.options.autoEncrypter) {
```

Comment:
Since we're inside of `server` here, we should use `this` instead of `server`. Also, we know that `this.s` will be defined because it is created in the constructor so if `this` is defined, `this.s` will also be. That leaves us to check `this.s.options` and `this.s.options.autoEncrypter`.